compiler/crates/relay-lsp/src/completion.rs

```diff
@@ -868,13 +870,31 @@ fn completion_items_for_request(
     }
 }
 
-fn resolve_completion_items_typename(type_: Type, schema: &SDLSchema) -> Vec<CompletionItem> {
+fn resolve_special_completion_items(type_: Type, schema: &SDLSchema) -> Vec<CompletionItem> {
+    let __id_item = CompletionItem {
+        label: "__id".to_owned(),
```

Comment:
I wonder if this should be exposed by the SDLSchema? A little nervous having this fieldname and type hard coded in multiple modules.